src/bots/zoom/src/index.ts

```diff
@@ -48,20 +48,7 @@ if (
   const browser = await launch({
     executablePath: puppeteer.executablePath(),
     headless: "new",
-    // slowMo: 10,
-    args: [
-        "--no-sandbox",
-    //     "--disable-setuid-sandbox",
-    //     "--disable-dev-shm-usage",
-    //     "--disable-gpu",
-        // "--use-fake-ui-for-media-stream",
-        "--use-fake-device-for-media-stream",
-        "--window-size=1920,1080",
-    //     "--disable-web-security",
-    //     "--allow-running-insecure-content",
-    //     "--autoplay-policy=no-user-gesture-required",
-    ],
-    // ignoreDefaultArgs: ["--mute-audio"],
+    args: ["--no-sandbox"],
```

Comment:
we should fix the permissions in the dockerfile so that we don't need `"--no-sandbox"`